### ISSUE TITLE
Audible Instruments Random Sampler: Pent quantizer error

### ISSUE THREAD

**Issue by veryfungi** [VCVRack/AudibleInstruments#73]
I believe there is an error on the Pent quantizer notes. They correct notes are C, D, E, G, A, C. The module is generating the notes C, D, F, G, A, C. Thank you.

**Comment by cschol** [comment]
You are correct about the Pentatonic notes, but the `Audible Instruments` version matches the behavior in the hardware module:
https://github.com/pichenettes/eurorack/blob/master/marbles/settings.cc#L80

Perhaps you can file an issue with Mutable Instruments.

**Comment by AndrewBelt** [comment]
Interesting, are those notes in a pentatonic scale with a root note other than C?

**Comment by anlexmatos** [comment]
> Interesting, are those notes in a pentatonic scale with a root note other than C?

Yes, that's the F Pentatonic scale.

**Comment by xandramax** [comment]
Yes, that's the F Pentatonic scale.
